packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -40,6 +42,22 @@ export function pluginReactAlias(options: Options): RsbuildPlugin {
           paths: [rootPath ?? api.context.rootPath],
         }),
       )
+
+      const pkgPath = path.join(reactLynxDir, 'package.json')
+      const pkgContent = readFileSync(pkgPath, 'utf-8')
+
+      let version: string | undefined
+      try {
+        const pkg = JSON.parse(pkgContent) as { version?: string }
+        version = semver.coerce(pkg?.version ?? undefined)?.version
```

Comment:
I don't think we need `semver.coerce` explictly. The `gte` can receive the `string` as a param.